scripts/lib/sign-win32.ts

```diff
@@ -52,39 +52,47 @@ export async function sign(workDir: string) {
 
   // Sign individual files.  See https://github.com/electron-userland/electron-builder/issues/5968
   const unpackedDir = path.join(workDir, 'unpacked');
-  const internalDir = 'resources/resources/win32/internal';
-
-  // make privileged-service.exe available to the instller during signing
+  const resourcesRootDir = 'resources/resources/win32';
+  const internalDir = path.join(resourcesRootDir, 'internal');
+  const binDir = path.join(resourcesRootDir, 'bin');
+  const whiteList: Record<string, Array<string>> = {
+    '.':                ['Rancher Desktop.exe'],
+    [resourcesRootDir]: ['wsl-helper.exe'],
+    [internalDir]:      ['host-resolver.exe', 'privileged-service.exe', 'steve.exe', 'vtunnel.exe'],
+    [binDir]:           ['docker.exe', 'docker-credential-none.exe', 'kuberlr.exe', 'nerdctl.exe', 'rdctl.exe'],
+  };
+
+  // make privileged-service.exe available to the installer during signing
   const privilegedServiceFile = 'privileged-service.exe';
   const privilegedServiceFrom = path.join(unpackedDir, internalDir, privilegedServiceFile);
   const privilegedServiceTo = path.join(process.cwd(), 'resources/win32/internal', privilegedServiceFile);
 
   await fs.promises.copyFile(privilegedServiceFrom, privilegedServiceTo);
```

Comment:
We will need to sign this copy of `privileged-service.exe` too (it shows up on uninstall).

Reply:
> kubectl.exe needs to be signed. (It's a copy of kuberlr.)

Does it? IIUC we are downloading the binary from Flavio's Github releases page, and don't build it with our CI. As a matter of policy we should only sign binaries where we have full control over the build and release process. Or am I missing something?

Why would we sign `kuberlr` but not `helm`?

Reply:
That's a fun question! I was assuming that "we" here was "SUSE", so it's still ours. If we decide that we _shouldn't_ sign it, then we shouldn't sign `kuberlr` either (and not just `kubectl`. Or we should build `kuberlr` ourselves.

As it is, though, it's at a minimum inconsistent.

Reply:
> we shouldn't sign kuberlr either (and not just kubectl. Or we should build kuberlr ourselves

Yes, I think we should sign neither of them (as one is just a copy of the other), and we should eventually build it ourselves.

>  I was assuming that "we" here was "SUSE"

I think it applies particularly to everything built by the processes controlled by the Rancher Desktop team.

Yes, Flavio works for SUSE, but it is his personal Github account that hosts it, so is this a release that should be signed by us? In that case you could argue that Matt also works for SUSE, so why not sign "his" helm releases as well?

It is not directly comparable, but I don't know how to draw a line clearly, which is why I would rather not sign something unless it is build directly by the project.

Reply:
ensured the uninstaller version of `privileged-service.exe` is signed, pulled out `kuberlr.exe`, didn't add `kubectl.exe`